test/e2e/storage/vsphere/pvc_label_selector.go

```diff
@@ -105,9 +105,8 @@ var _ = utils.SIGDescribe("PersistentVolumes [Feature:LabelSelector]", func() {
 })
 
 func testSetupVSpherePVClabelselector(c clientset.Interface, nodeInfo *NodeInfo, ns string, ssdlabels map[string]string, vvollabels map[string]string) (volumePath string, pv_ssd *v1.PersistentVolume, pvc_ssd *v1.PersistentVolumeClaim, pvc_vvol *v1.PersistentVolumeClaim, err error) {
```

Comment:
Does err need to be passed in as an argument at all here? It seems to be immediately overwritten by `nodeInfo.VSphere.CreateVolume(...)`.

cc @hoegaarden

Reply:
I think so,  err  is redundant ,I will remove it

Reply:
error is as a  return value , I think no need to do anything

Reply:
I agree with tanshanshan here.
That seems we forgot to remove the Expect(err) in https://github.com/kubernetes/kubernetes/commit/c0490fa62365a37d1a4c3c27b746822782849fc2#diff-0714d41e8eaadd3eb93bba91a7347782R114